src/Command/SyncThumbsCommand.php

```diff
@@ -141,7 +141,7 @@ public function execute(InputInterface $input, OutputInterface $output)
             }
 
             //clear entity manager for saving memory
-            $this->getMediaManager()->getEntityManager()->clear();
+            $this->getMediaManager()->getObjectManager()->clear();
```

Comment:
It will stop working with orm, so this is not a good fix, Try using some method in a common interface

Reply:
Why? ORM media manager extend the same BaseManager as ODM and PHPCR.

Reply:
The why does not work with entityManager?

Reply:
@greg0ire here is another usage of a method not in the common interface

Reply:
:sob: